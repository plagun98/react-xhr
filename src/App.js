import React from 'react';
import 'bootstrap/dist/css/bootstrap.min.css';
import List from './components/List';

const App = () => {
  return (
    <div>
      <List/>
    </div>
  );
}

export default App;
