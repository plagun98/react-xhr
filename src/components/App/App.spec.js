import React from 'react';
import App from './App';

import { shallow } from 'enzyme';

describe('App container initial', () => {
    
    const appContainer = shallow(<App.WrappedComponent/>)

    it('renders properly', () => {
        expect(appContainer).toMatchSnapshot()
    })
})
