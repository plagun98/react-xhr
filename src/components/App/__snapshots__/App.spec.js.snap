// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App container initial renders properly 1`] = `ShallowWrapper {}`;
